scripts/node/icons/helpers/svgoConfig.js

```diff
@@ -36,5 +36,5 @@ exports.svgoPlugins = [
   { convertShapeToPath: false },
   { sortAttrs: true },
   { removeDimensions: true },
-  { removeAttrs: { attrs: '(stroke|fill)' } },
+  { removeAttrs: { attrs: '(stroke|fill|fill-opacity|filter)' } },
```

Comment:
Now that `svgr` no longer does optimization for us, we want to make sure that the files in `lib/icons/svg` do not contain attributes that could apply unwanted styles to elements.

The nice thing here is that the icons build only needs to run an optimize step once. The component build reads directly from the optimized files in `lib` now.